commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/arguments/CustomArgument.java

```diff
@@ -22,6 +22,7 @@
 
 import java.io.Serializable;
 
+import net.kyori.adventure.text.TextComponent;
```

Comment:
This import was added, but it doesn't seem to be used. I'm guessing it's just leftover from some past experiments.
```suggestion
```